BAZProject/RemoteData/LoadImages.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  LoadImages.swift
+//  BAZProject
+//
+//  Created by avirgilio on 26/01/23.
+//
+import UIKit
+
+class LoadRemotedata{
+    /**
+     - The `saveImages(from movies: [ResultMovie])` load from the URL `https://image.tmdb.org/t/p/w500`
+     - the images from the atributte `posterPath` in the `movies` object.
+     - The image is saved locally to avoid the consume of images service more than the needed times
+     */
+    
+    func saveImages(from movies: [ResultMovie]){
+        var imageResult = UIImage()
+        var totalPath = ""
+        movies.forEach { movie in
+            totalPath = filePath()?.appendingPathComponent(movie.posterPath).absoluteString ?? ""
+            guard let url = URL(string: "https://image.tmdb.org/t/p/w500\(movie.posterPath)") else {
+                        return
+                    }
+            if FileManager.default.fileExists(atPath: totalPath ){return}
+            let dataTask = URLSession.shared.dataTask(with: url) {  (data, _, _) in
+                if let data = data {
+                    imageResult = UIImage(data: data) ?? UIImage()
+                    self.saveImgToLocal(imageResult, path: movie.posterPath)
+                }
+            }
+            dataTask.resume()
+        }
+    }
+    private func saveImgToLocal(_ image: UIImage, path: String){
+        let pathFolder = path
+        let totalPath = filePath()?.appendingPathComponent(String(pathFolder.dropFirst()))
+        guard let data = image.jpegData(compressionQuality: 0.5) else { return }
+        if !FileManager.default.fileExists(atPath: totalPath?.absoluteString ?? ""){
+            if let path = totalPath {
+                do {
+                        try? data.write(to: path)
+                    }
+            }
+        }
+    }
+    
+    func loadImgsFromLocal(strPath: String) -> UIImage{
```

Comment:
Busca que tus funciones se lean con facilidad, escribir un código limpio [Ref: 
`https://www.amazon.com.mx/Clean-Code-Handbook-Software-Craftsmanship/dp/0132350882`
] nos pide que los nombres sean pronunciables en su totalidad, sin importar que seamos capaces de inferir que un imgs = images, o que un str = string

```suggestion
    func loadImagesFromLocal(stringPath: String) -> UIImage {
```